test/generators/csharp/presets/__snapshots__/NewtonsoftSerializerPreset.spec.ts.snap

```diff
@@ -52,13 +52,13 @@ public class TestConverter : JsonConverter<Test>
   value.StringProp = jo[\\"string prop\\"].ToObject<string>(serializer);
 }
 if(jo[\\"numberProp\\"] != null) {
-  value.NumberProp = jo[\\"numberProp\\"].ToObject<double>(serializer);
+  value.NumberProp = jo[\\"numberProp\\"].ToObject<double?>(serializer);
 }
 if(jo[\\"enumProp\\"] != null) {
-  value.EnumProp = EnumTestExtensions.ToEnumTest(jo[\\"enumProp\\"]);
+  value.EnumProp = EnumTest?Extensions.ToEnumTest?(jo[\\"enumProp\\"]);
```

Comment:
Same as the other comment, need to change the following lines: 
- https://github.com/asyncapi/modelina/blob/3cfd68b877d04f48c492b1eb53cf58b1c2d7179d/src/generators/csharp/presets/NewtonsoftSerializerPreset.ts#L19
- https://github.com/asyncapi/modelina/blob/3cfd68b877d04f48c492b1eb53cf58b1c2d7179d/src/generators/csharp/presets/NewtonsoftSerializerPreset.ts#L71